ansible/templates/whisk.properties.j2

```diff
@@ -7,6 +7,7 @@ testing.auth={{ openwhisk_home }}/ansible/files/auth.guest
 vcap.services.file=
 
 whisk.logs.dir={{ whisk_logs_dir }}
+whisk.coverage.logs.dir={{ coverage_logs_dir }}
```

Comment:
is this used anywhere?

Reply:
This is used in [tests/build.gradle][1] while generating coverage report. The task would move coverage files from this directory to respective modules

[1]: https://github.com/apache/incubator-openwhisk/pull/3685/files#diff-f38475a309b117418ec21c944e4d60ae